frontend/src/components/SingleChat.js

```diff
@@ -221,35 +217,34 @@ const SingleChat = ({ fetchAgain, setFetchAgain }) => {
             )}
 
             <FormControl
-              onKeyDown={sendMessage}
+              onKeyDown={(event) => event.key === "Enter" && sendMessage()}
```

Comment:
verify the key down event in sendMessage function

Reply:
The original code did not have a send button. By pressing Enter, the message was sent, but it didn't work on mobile screens, so I added a new send button. @kartikayasijaa

Reply:
Means in large screen ??? no

Reply:
Both have send button